pkg/ebpftracer/signature/git_clone_detected_test.go

```diff
@@ -0,0 +1,89 @@
+package signature
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindCloneAndRepository(t *testing.T) {
+	type testCase struct {
+		name               string
+		args               []string
+		expectedRepo       string
+		expectedCloneFound bool
+	}
+
+	testCases := []testCase{
```

Comment:
I can imagine users crafting a rule that triggers an anomaly when a repository is cloned from a curated list of malicious repositories. We could also have this as a default rule.

I am thinking about the format of this rule. Should we store the git repository in the event without the transport protocol and add a new field with the protocol? And also the server, something like:

```proto
message GitCloneDetectedFinding {
  string server = 1;
  string repo = 2;
  string protocol = 2;
}
```

So we can have the following events:

```json
{
  "server": "github.com",
  "repo": "castai/kvisor",
  "protocol": "ssh"
},
{
  "server": "gitlab.com",
  "repo": "gitlab-org/api/client-go",
  "protocol": "https"
}
```

Reply:
Good point! Extended the protocol.

Reply:
Awesome!